From 908a893055018d280171d3792388ca435836c00a Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Fri, 31 Mar 2017 13:40:00 +0800
Subject: [PATCH 28/65] do not support manually set time

Since NTP setting is not supported in anaconda, it does not
support set time manually neither.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

Rebase for anaconda 34.

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 pyanaconda/ui/gui/spokes/datetime_spoke.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/pyanaconda/ui/gui/spokes/datetime_spoke.py b/pyanaconda/ui/gui/spokes/datetime_spoke.py
index 58050c0b3..ff4b69577 100644
--- a/pyanaconda/ui/gui/spokes/datetime_spoke.py
+++ b/pyanaconda/ui/gui/spokes/datetime_spoke.py
@@ -542,6 +542,9 @@ class DatetimeSpoke(FirstbootSpokeMixIn, NormalSpoke):
                              _("Restoring hardware time..."))
             threadMgr.wait(constants.THREAD_TIME_INIT)
 
+        # Do not support manually set time
+        self._set_date_time_setting_sensitive(False)
+
         hubQ.send_ready(self.__class__.__name__)
 
         # report that we are done
-- 
2.7.4

